### ISSUE TITLE
[FLUTTER WEB] PdfPreview works in debug mode, but not in release mode

### ISSUE THREAD

**Issue by adiomas** [DavBfr/dart_pdf#1169]
**Description**
Pdf is working in debug mode(localhost), but on hosting I get an error "Unable to display the document". (I added scripts in index.html). Same behaviour is on desktop web or mobile browser (Safari, Chrome)

**Code snippet**
```dart
Stack(
          children: [
            PdfPreview(
              pdfFileName: 'Pdf',
              build: (_) => pdf,
              padding: const EdgeInsets.only(top: 30),
              canChangeOrientation: false,
              useActions: false,
              canChangePageFormat: false,
            ),
            const Positioned(
              top: 30,
              left: 8,
              child: AppointmentsBackButton(),
            ),
          ],
        ),
```

**Expected behavior**
This is screenshot from debug mode(localhost).
![](https://i.ibb.co/sF6dm8n/Screenshot-2022-10-14-at-11-56-24.png)

**Current behavior**
This is screenshot from release mode(on host)
![](https://i.ibb.co/KyTz0h9/Screenshot-2022-10-14-at-11-47-05.png)

**Flutter Doctor**
```
[✓] Flutter (Channel dev, 3.3.0-0.5.pre, on macOS 12.4 21F79 darwin-arm (Rosetta), locale en-HR)
    • Flutter version 3.3.0-0.5.pre on channel dev at /Users/qguest/flutter/versions/3.3.1-stable
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 096162697a (7 weeks ago), 2022-08-22 15:34:14 -0700
    • Engine revision ad3d868e0d
    • Dart version 2.18.0 (build 2.18.0-271.7.beta)
    • DevTools version 2.15.0

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0-rc3)
    • Android SDK at /Users/qguest/Library/Android/sdk
    • Platform android-33, build-tools 31.0.0-rc3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.4.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 13F100
    • CocoaPods version 1.11.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)

[✓] VS Code (version 1.72.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.50.0

[✓] Connected device (2 available)
    • macOS (desktop) • macos  • darwin-arm64   • macOS 12.4 21F79 darwin-arm (Rosetta)
    • Chrome (web)    • chrome • web-javascript • Google Chrome 106.0.5249.119

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```

**Desktop (please complete the following information):**
 - [ ] iOS
 - [ ] Android
 - [x] Browser
 - [ ] Windows
 - [ ] Linux

**Smartphone (please complete the following information):**
- Browser [Safari,Chrome]

**Comment by alexandersidum** [comment]
Faced the same issue

**Comment by Czapelxd** [comment]
Hey, I am fancying the same issue but the 'flutter clean' is not fixing it. In debug works smoothly but in release I am getting NoSuchMethodError: method not found: 'a2' (a.a2 is not a function)

**Comment by DavBfr** [comment]
Fixed in #1176
